### PR TITLE
doc: Updates nodejs-express stack readme

### PR DESCRIPTION
**Pull request by neeraj-laad**
docs: Updates nodejs-express stack readme

### CODE REVIEW

File: incubator/nodejs-express/README.md [hunk 1]
```diff
@@ -1,79 +1,65 @@
 # Node.js Express Stack
 
-This provides a prototype platform definition for Node.js Express based projects. This prototype uses Node.js v10 and Express v4.16.0
+The Node.js Express stack extends the [Node.js stack](../nodejs/README.md) and provides a consistent way of developing web applications using [Express](https://expressjs.com). Express is a fast, minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications.
 
-## Stacks
+This stack is based on `Node.js v10` and `Express v4.16.0` and includes enables health checking and application metrics out of the box. The stack also provides a set of tests written using [Mocha](https://mochajs.org) test framework while the application metadata is provided via a `package.json` file.
```
Comment (helenmasters) [inline incubator/nodejs-express/README.md]: include enables?

Comment (helenmasters) [reply incubator/nodejs-express/README.md]: Adding the and a , helps:

The stack also provides a set of tests written using the [Mocha](https://mochajs.org) test framework, while the application metadata is provided via a `package.json` file.

File: incubator/nodejs-express/README.md [hunk 2]
```diff
@@ -1,79 +1,65 @@
...
-This stack also comes with prometheus metrics, which has been preconfigured to work with your applicaiton. You will not be able to override this endpoint:
+This stack also comes with Prometheus metrics, which has been preconfigured to work with your application. You will not be able to override this endpoint:
 
-- Metrics: http://localhost:3000/metrics
+- Metrics endpoint: http://localhost:3000/metrics
 
 ## Templates
 
-A template is what you will use as a template to begin your developement. When initializing your project you have the option to choose between using the hello-world application or the hello-express application. This stack consists of two nodejs-express templates, A hello-world and a hello-express template.
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the simplest Node.js Express application you can write.
```
Comment (helenmasters) [inline incubator/nodejs-express/README.md]: Can you move templates up to the top as per the other Stack READMEs?

File: incubator/nodejs-express/README.md [hunk 3]
```diff
@@ -1,79 +1,65 @@
...
-```bash
-cd incubator/nodejs-express/stack
-docker build -t nodejs-express -f Dockerfile-stack .
-```
-3. Run your application
-```bash
-appsody run
-```
\ No newline at end of file
+Health endpoint: http://localhost:3000/health
```
Comment (helenmasters) [inline incubator/nodejs-express/README.md]: These are displaying horizontally across the page not in the list. Also can you put the Application endpoint first, as we didn't explicitly put this in the Getting Started?

File: incubator/nodejs-express/README.md [hunk 4]
```diff
@@ -1,79 +1,65 @@
...
 
-- Readiness Check: http://localhost:3000/ready
-- Liveness Check: http://localhost:3000/live
-- Health Check: http://localhost:3000/health
+You can override or enhance the following endpoints by configuring your own health checks in your application.
 
-### Application metrics
+## Application metrics
 
 Enable powerful monitoring for your distributed application and configure rule-based alerting using Prometheus open source project. This is vital for diagnosing problems and ensuring the reliability of your application.
```
Comment (helenmasters) [inline incubator/nodejs-express/README.md]: "using the Prometheus open source project" or "using Prometheus".

File: incubator/nodejs-express/README.md (hunk 2 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: In this case the stack includes these capabilities not the template.. hence the template section is further down.

File: incubator/nodejs-express/README.md (hunk 3 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: sorted formatting and order

File: incubator/nodejs-express/README.md (hunk 4 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: addressed

File: incubator/nodejs-express/README.md (hunk 3 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: resolved

File: incubator/nodejs-express/README.md (hunk 2 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: this ones had more capabilities in the stack. hence the first sections looks bigger..

File: incubator/nodejs-express/README.md (hunk 1 above)
Comment (neeraj-laad) [reply incubator/nodejs-express/README.md]: changed

File: incubator/nodejs-express/README.md (hunk 2 above)
Comment (helenmasters) [reply incubator/nodejs-express/README.md]: Ah ok, it wasn't obvious to me that the stack contained those capabilities.

**Pull request merged**